aws.go

```diff
@@ -118,9 +118,13 @@ func scrape(collector *cwCollector, ch chan<- prometheus.Metric) {
 		totalRequests.Inc()
 
 		if err != nil {
+			collector.ErroneousRequests.Inc()
 			fmt.Println(err)
 			continue
 		}
+		else {
```

Comment:
should be
```
} else {
```

Reply:
Else block is redundant and could be removed altogether.